test/contrib/rails/database_test.rb

```diff
@@ -1,4 +1,5 @@
 require 'helper'
+require 'pry'
```

Comment:
I think we should remove it, right?

Reply:
Oh whoops, totally.

Reply:
No worries, reviews are meant for that 💯